x/auth/ante.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
```

Comment:
should this alias be `authtypes` for consistency?

Reply:
no because this is within the auth package